### ISSUE TITLE
Cookie Issue for Self Service Login - Timezone Issue

### ISSUE THREAD

**Issue by danieldjewell** [LinOTP/LinOTP#84]
Just upgraded from 2.9 to 2.10.0.2-1 on debian jessie using the APT repository. 

When trying to login to /selfservice/, the Set-Cookie HTTP header returned specifies an expiry time that is [Local Time] + 1 hour ... but in GMT/UTC. For example, if the local time on the server is 11:00:00, the cookie returned is 12:00:00 GMT (regardless of the timezone). 

If the timezone is UTC-07:00, for example, the cookie expiry time will be 6 hours in the past because timezone differences aren't being taken into account. 

Based on what I'm seeing, this issue would only affect users west of GMT/UTC. All users in Europe would never notice this problem (as the UK is UTC or UTC+01:00 and Central Europe/DE/CH/etc. is UTC+01:00 in the winter and UTC+02:00 in the summer). This could also mean that for users in very Eastern timezones (e.g. Asia/Australia), the cookie being set could be set to expire much longer than 1 hour. 

Not 100% sure what's causing this - sorry, I didn't have enough time to go back and compare 2.9 to 2.10 -- however, I was able to fix in a relatively simple way. See below. 

# Simple Patch

I wrote a simple patch in lib/userservice.py ... (I didn't make a commit because I'm unsure if this is really the best way to address, as above) 

https://github.com/LinOTP/LinOTP/blob/6614e7e306412daf463f84fef67695256d29e93d/linotpd/src/linotp/lib/userservice.py#L155-L163

```python
import time
#....
#the datetime.now() method returns the local time on the system 
#the datetime.utcnow() method would return UTC time
#time.timezone contains the number of seconds of offset (+ for west, - for east) -- just add it to the delta
delta = datetime.timedelta(seconds=(1 * 60 * 60) + time.timezone) 
```

**Comment by danieldjewell** [comment]
As a secondary comment to this issue. In general, the way that time is handled within linotp doesn't really take into account timezones. 

Aber es gibt mehr als eine Zeitzone in der Welt...

**Comment by lukengda** [comment]
We fixed #136 which could improve on your problem. Can you report, if your problem still persists with the 2.11.2 release?

**Comment by lukengda** [comment]
Will close the issue here, as we now take time zone shift into account with the latest release: http://linotp.org/resources/changelogs.html
